uqcsbot/api.py

```diff
@@ -208,6 +208,16 @@ def _add_channel(self, chan_dict: dict) -> Channel:
         self._channels_by_id[chan.id] = chan
         return chan
 
+    def reload_ims(self, *args):
```

Comment:
provide a docstring saying what this function does and what an `im` is-- its not immediately clear :^)

Reply:
also, given my other comment, I think this function can just be moved inline to where channels are added.